packages/pelagos/src/treeView/TreeView.js

```diff
@@ -6,15 +6,17 @@ import TreeNodeContext from './TreeNodeContext';
 
 import './TreeView.less';
 
-const nodeContextValue = {level: 0, padding: 0};
+const nodeContextValue = {level: 0, padding: 0, parentPath: []};
+
+const getFocusedId = (selected, children) => (selected ? selected.at(-1) : Children.toArray(children)[0]?.props.id);
 
 /** A component implementing the tree view pattern. */
 const TreeView = ({className, size, selected, children, onSelect, ...props}) => {
```

Comment:
I think this needs to be updated in `TreeView.propTypes`?
```
	/** The ID of the selected node. */
	selected: PropTypes.string,
```

Leaving the comment here because github won't let me comment on line 43.

Reply:
Yes, thank you for catching it